stock_by_warehouse/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License: LGPL-3
+
+Functional Rationale:
+---------------------
+
+Provide a nice widget in the product form to see the quantity of product
+by warehouse in order to help you to review such information as fast as possible.
+
+.. figure:: static/description/widget_by_warehouse.png
```

Comment:
I think it should be great if both of the images (here and the one with locations) were updated so if there is a change in the form the user can see it and not look for a field that was moved/removed